playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+# Remote Control
+
+The objective of this workshop is to learn to wield powerful
+[APIs](http://www.quora.com/What-is-an-API) with minimal code.
+
+This tutorial can be used by complete beginners or more advanced programmers.
+
+**For Beginners**
+
+You will learn how to do really cool things with really simple code. You won't
+understand how it all works and that's totally cool. Your objective is to learn
+how to make things happen. Later you can come back and figure out how it all
+works.
+
+**For More Experienced Folk**
+
+This tutorial will likely show you things that you didn't know that you could
+do. I'd recommend skipping around in the tutorial and find what strikes your
+interest.
+
+## Partners
+
+Before starting this workshop, find a partner! You'll be pair
+programming!
+
+## Quick Demo
+
+We're going to write some code that calls your phone and speaks some
+text.
+
+Ready?
+
+Open [this JS Bin](http://jsbin.com/papawo/6/edit?html,js,console). Leave this
+tab open for the rest of the tutorial.
+
+JS Bin is a website to easily write code for websites.
+
+### Adding your Twilio sid and token
+
+We are going to use a service called Twilio to do things like send text messages
+and make phone calls.
+
+To add this capability to any HTML file, just add this script tag inside the
+bottom of your body tag:
+
+```html
+<script src="//bit.ly/twilio-basic-v1"
+  sid="YOUR_TWILIO_ACCOUNT_SID_HERE"
+  token="YOUR_TWILIO_AUTH_TOKEN_HERE"
```

Comment:
@zachlatta I know this is a thing (
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes),
however I suggest we leave it as it is for simplicity.
Do you know which browsers do not support the current method?

On Wed, 7 Oct 2015 01:09 Zach Latta <notifications@github.com> wrote:

> In playbook/workshops/remote_control/README.md
> <https://github.com/hackedu/hackedu/pull/234#discussion_r41268245>:
>
> > +tab open for the rest of the tutorial.
> > +
> > +JS Bin is a website to easily write code for websites.
> > +
> > +### Adding your Twilio sid and token
> > +
> > +We are going to use a service called Twilio to do things like send text messages
> > +and make phone calls.
> > +
> > +To add this capability to any HTML file, just add this script tag inside the
> > +bottom of your body tag:
> > +
> > +```html
> > +<script src="//bit.ly/twilio-basic-v1"
> > +  sid="YOUR_TWILIO_ACCOUNT_SID_HERE"
> > +  token="YOUR_TWILIO_AUTH_TOKEN_HERE"
>
> Shouldn't these be data-sid and data-token to conform to JavaScript
> standards?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/234/files#r41268245>.
>